packages/create-itp-react-app/README.md

```diff
@@ -1,7 +1,7 @@
 ## Generated project structure
 
 ```
-├── docs
+├── design-docs
```

Comment:
- [x] could you update or remove the '##docz' section as well?

Reply:
nvm, fixed here: https://github.com/inthepocket/itp-react-components/pull/10/files#diff-2fa31966e46a846a4dab0a8e40a8145dL32